portfolio/src/main/webapp/script/comment-script.js

```diff
@@ -12,12 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Spaces out comment text from author name. */
+const commentHyphen = " -";
+
 /**
  * Fetches data from servlet and sets it in the comments section of portfolio.
  * Called whenever comments section is loaded.
  */
 async function getData() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('data-container').innerText = quote;
+  const jsonData = await response.json();
+  
+  const commentsListElement = document.getElementById('comments-container');
+  commentsListElement.innerHTML = '';
+  for (comment of jsonData) {
+    commentsListElement.appendChild(createListElement(comment.commentText + commentHyphen + comment.name));
+  }
+}
+
+/** 
+ * Creates an <li> element containing text from comment. 
+ * @param {String} text Text to be placed in new element.
+ * @return {<li>} liElement Element to be appended in comments section of portfolio.
+ */
+function createListElement(text) {
```

Comment:
is there a typo in the name of the method? Should it be createLiElement?

Reply:
ListElement = `<li>`